tests/client_tests/stock_jdbc/src/test/java/stock_jdbc/JdbcMetaDataTest.java

```diff
@@ -23,7 +23,7 @@
 
     @ClassRule
     public static final CrateTestCluster TEST_CLUSTER = CrateTestCluster
-        .fromURL("https://cdn.crate.io/downloads/releases/nightly/crate-4.2.0-202002120002-289d32d.tar.gz")
+        .fromURL("https://cdn.crate.io/downloads/releases/nightly/crate-4.2.0-202003080002-5ccd5ca.tar.gz")
```

Comment:
I tried to point this to latest-nighlty, but the download does not follow the redirect and i don't get the latest version.